src/interfaces.ts

```diff
@@ -108,37 +108,53 @@ export interface Templates {
   [index: string]: string | Templates;
 }
 
+export interface FiddleEditorTypes {
```

Comment:
_nit_ Maybe this could be called "EditorTypes"? The "Fiddle" prefix doesn't seem to add value